cmd/cli/handler_migrate.go

```diff
@@ -62,7 +63,7 @@ func (h *MigrateHandler) MigrateSQL(cmd *cobra.Command, args []string) {
 			return
 		}
 		viper.Set(configuration.ViperKeyDSN, args[0])
-		d = driver.NewDefaultDriver(logrusx.New(), false, nil, "", "", "", false)
+		d = driver.NewDefaultDriver(logrusx.New(), prometheus.NewRegistry(), false, nil, "", "", "", false)
```

Comment:
Is `prometheus.NewRegistry()` injected somewhere for testing? If not, this should probably come from the `registry_base` instead.

Reply:
Done! :)